### PR TITLE
Create glow_pcb_8266.yaml

### PR DESCRIPTION
**Pull request by adonno**
as per Paulus' request not to have Home-Assistant in the name i've given it the following name.  This change is the yaml for the all in one PCB version of Glow running on an ESP8266.

### CODE REVIEW

File: glow_pcb_8266.yaml [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+    id: glow_version
+    icon: "mdi:label-outline"
+    entity_category: diagnostic
+    lambda: |-
+      return {"${project_version}"};
+    update_interval: 6h
+  # ESPHome version
+  - platform: version
+    hide_timestamp: true
+    name: '${friendly_name} - ESPHome Version'
```
Comment (klaasnicolaas) [inline glow_pcb_8266.yaml:146]:
```suggestion
    name: '${friendly_name} ESPHome Version'
```

File: glow_pcb_8266.yaml [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+  # IP address and connected SSID
+  - platform: wifi_info
+    ip_address:
+      name: '${friendly_name} - IP Address'
+      icon: mdi:wifi
+    ssid:
+      name: '${friendly_name} - Connected SSID'
```
Comment (klaasnicolaas) [inline glow_pcb_8266.yaml:153]:
```suggestion
      name: '${friendly_name} Connected SSID'
```

File: glow_pcb_8266.yaml [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+      return {"${project_version}"};
+    update_interval: 6h
+  # ESPHome version
+  - platform: version
+    hide_timestamp: true
+    name: '${friendly_name} - ESPHome Version'
+  # IP address and connected SSID
+  - platform: wifi_info
+    ip_address:
+      name: '${friendly_name} - IP Address'
```
Comment (klaasnicolaas) [inline glow_pcb_8266.yaml:150]:
```suggestion
      name: '${friendly_name} IP Address'
```

File: glow_pcb_8266.yaml (hunk 2 above)
Comment (klaasnicolaas) [inline glow_pcb_8266.yaml:153]:
```suggestion
      name: '${friendly_name} Connected SSID'
```

File: glow_pcb_8266.yaml [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+      icon: mdi:wifi
+    ssid:
+      name: '${friendly_name} - Connected SSID'
+      icon: mdi:wifi-strength-2
+
+sensor:
+  # WiFi signal
+  - platform: wifi_signal
+    name: "${friendly_name} - WiFi Signal"
```
Comment (klaasnicolaas) [inline glow_pcb_8266.yaml:159]:
```suggestion
    name: "${friendly_name} WiFi Signal"
```

File: glow_pcb_8266.yaml [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+    update_interval: 120s
+
+  # Pulse meter
+  - platform: pulse_meter
+    name: '${friendly_name} - Power Consumption'
```
Comment (klaasnicolaas) [inline glow_pcb_8266.yaml:164]:
```suggestion
    name: '${friendly_name} Power Consumption'
```

File: glow_pcb_8266.yaml (hunk 4 above)
Comment (klaasnicolaas) [inline glow_pcb_8266.yaml:159]:
```suggestion
    name: "${friendly_name} WiFi Signal"
```

File: glow_pcb_8266.yaml (hunk 5 above)
Comment (klaasnicolaas) [inline glow_pcb_8266.yaml:164]:
```suggestion
    name: '${friendly_name} Power Consumption'
```

File: glow_pcb_8266.yaml [hunk 6]
```diff
@@ -0,0 +1,258 @@
...
+            red: 100%
+            green: 100%
+            blue: 100%
+            flash_length: 250ms 
+    filters:
+      # multiply value = (60 / imp value) * 1000
+      # - multiply: 60
+      - lambda: return x * ((60.0 / id(select_pulse_rate).state) * 1000.0);
+    total:
+      name: '${friendly_name} - Total Energy'
```
Comment (klaasnicolaas) [inline glow_pcb_8266.yaml:187]:
```suggestion
      name: '${friendly_name} Total Energy'
```

Comment (klaasnicolaas) [inline glow_pcb_8266.yaml:187]:
```suggestion
      name: '${friendly_name} Total Energy'
```

File: glow_pcb_8266.yaml [hunk 7]
```diff
@@ -0,0 +1,258 @@
...
+      state_class: total_increasing
+      device_class: energy
+      accuracy_decimals: 3
+      filters:
+        # multiply value = 1 / imp value
+        # - multiply: 0.001
+        - lambda: return x * (1.0 / id(select_pulse_rate).state);
+  # Total day useage
+  - platform: total_daily_energy
+    name: '${friendly_name} - Daily Energy'
```
Comment (klaasnicolaas) [inline glow_pcb_8266.yaml:200]:
```suggestion
    name: '${friendly_name} Daily Energy'
```

File: glow_pcb_8266.yaml [hunk 8]
```diff
@@ -0,0 +1,258 @@
...
+  status_led: GPIO2
+
+dashboard_import:
+  package_import_url: github://klaasnicolaas/home-assistant-glow/glow_pcb_8266.yaml
+
+esphome:
+  name: '${device_name}'
+  comment: '${device_description}'
+  project:
+    name: "klaasnicolaas.home-assistant-glow"
```
Comment (klaasnicolaas) [inline glow_pcb_8266.yaml:25]:
```suggestion
    name: "klaasnicolaas.hardware-glow-v1"
```
Or something similar?

File: glow_pcb_8266.yaml [hunk 9]
```diff
@@ -0,0 +1,258 @@
...
+    optimistic: true
+    mode: box
+    min_value: 100
+    max_value: 10000
+    step: 100
+    restore_value: yes
+    initial_value: 1000
+
+light:
+- platform: neopixelbus
```
[the remark this answers is not in the document]
Comment (adonno) [reply glow_pcb_8266.yaml:98]: let them play with it :-D

**Comment by klaasnicolaas** [comment]
Let's have a talk soon, since there are a lot of changes going on with the project, it might also be a good time to take up the Glow PCB further 😉

**Pull request closed without merging**